src/main/kotlin/no/nav/tiltakspenger/arena/repository/ArenaVedtakDTO.kt

```diff
@@ -23,4 +23,22 @@ data class ArenaVedtakDTO(
 ) {
     fun fomGyldighetsdato(): LocalDateTime? = (fomVedtaksperiode ?: registrertDato)!!.atStartOfDay()
     fun tomGyldighetsdato(): LocalDateTime? = tomVedtaksperiode?.atStartOfDay()
+
+    fun isTiltakspenger(): Boolean = this.rettighettype == ArenaRettighet.BASI
```

Comment:
Stusset litt på denne, men ser at ‘Tiltakspenger' het ‘Basisytelse' før 2014. Så da gir det mening 👌 
Synd de ikke har endret på det så det blir litt verre for oss å holde orden på 😓

Reply:
Ja, det er jo Arena-kode det jobbes med her, så vi må forholde oss til en del av deres ord og begreper..